Rubberduck.Core/App.cs

```diff
@@ -182,24 +183,22 @@ private void ApplyCultureConfig()
 
         private static void LocalizeResources(CultureInfo culture)
         {
-            //TODO: this method needs something better - maybe use reflection to discover all resourcees
-            //      to set culture for all resources files?
-            Resources.RubberduckUI.Culture = culture;
-            Resources.About.AboutUI.Culture = culture;
-            Resources.Inspections.InspectionInfo.Culture = culture;
-            Resources.Inspections.InspectionNames.Culture = culture;
-            Resources.Inspections.InspectionResults.Culture = culture;
-            Resources.Inspections.InspectionsUI.Culture = culture;
-            Resources.Inspections.QuickFixes.Culture = culture;
-            Resources.Menus.RubberduckMenus.Culture = culture;
-            Resources.RegexAssistant.RegexAssistantUI.Culture = culture;
-            Resources.Settings.SettingsUI.Culture = culture;
-            Resources.Settings.ToDoExplorerPage.Culture = culture;
-            Resources.Settings.UnitTestingPage.Culture = culture;
-            Resources.ToDoExplorer.ToDoExplorerUI.Culture = culture;
-            Resources.UnitTesting.AssertMessages.Culture = culture;
-            Resources.UnitTesting.TestExplorer.Culture = culture;
-            Resources.Templates.Culture = culture;
+            var localizers = AppDomain.CurrentDomain.GetAssemblies()
+                .SingleOrDefault(assembly => assembly.GetName().Name == "Rubberduck.Resources")
```

Comment:
If we know the assembly already, shouldn't we reflect the assembly by a type it contains instead of iterating over all assemblies loaded in the current domain?

Reply:
Like, `typeof(Resources.RubberduckUI).Assembly.DefinedTypes.SelectMany(...)` - agreed, less stringly-typed, too.